silk-core/build.gradle.kts

```diff
@@ -11,6 +12,12 @@ plugins {
     kotlin("plugin.serialization")
 }
 
+dependencies {
+    modApi("net.fabricmc.fabric-api:fabric-api:$fabricApiVersion")
+
+    modCompileOnly("me.lucko:fabric-permissions-api:0.3.1")
```

Comment:
Im pretty unsure about this. Iirc silk was designed to be standalone. Depending on a 3rd party library sounds against this

Reply:
hmm, that is fair

should I move it to a fabric-specific module?

but, the thing, is, the two platforms that silk is targeting (fabric & paper) both have permissions apis.
however, at least to my knowledge, they don't share an api. so that is kinda annoying and smth that would need to be worked around.

for now, I can remove it

Reply:
I'm not the one who is in position to tell you what we should add to silk xd

Reply:
I think that a permission module could be a possible module for silk or at least a part of silk-core

Reply:
yeah, though an entire module *just* for a permissions api feels a bit much, yk?

which is why I added it as a compile-only dep